src/components/Navigation/Navigation.jsx

```diff
@@ -39,14 +39,17 @@ export default class Navigation extends React.Component {
 
           <div className="navigation__search">
             <input
-              type="text"
+              aria-label="Search documentation"
+              type="search"
```

Comment:
Usually the type `search` adds some extra styles or autocompletes that might not go well with the styles we have now, have you tested this change in different browsers?

Reply:
Wow. Spotted some strange artifact on Safari 11. Thank you. Will figure out how to avoid it.